src/main/java/io/spaship/operator/service/k8s/Operator.java

```diff
@@ -37,414 +36,413 @@
 // todo this class violates single responsibility principal
 @ApplicationScoped
 public class Operator implements Operations {
-  private static final Logger LOG = LoggerFactory.getLogger(Operator.class);
-  private static final String MANAGED_BY = "managedBy";
-  private static final String WEBSITE = "website";
-  private static final String ENVIRONMENT = "environment";
-  private static final String SPASHIP = "spaship";
-  private final OpenShiftClient ocClient;
-  private final EventManager eventManager;
-  private final String domain;
-  private final String routerDomain;
-  private final String routerShard;
-  private final String appInstance;
-  private final String deDebugNs;
-
-  public Operator(@Named("default")OpenShiftClient ocClient,
-      EventManager eventManager, @Named("deNamespace") String ns) {
-    this.ocClient = ocClient;
-    this.eventManager = eventManager;
-    domain = ConfigProvider.getConfig().getValue("operator.domain.name", String.class);
-    routerDomain = ConfigProvider.getConfig().getValue("operator.router.domain.name", String.class);
-    routerShard = ConfigProvider.getConfig().getValue("operator.router.shard.type", String.class);
-    appInstance = setAppInstanceValue();
-    this.deDebugNs = ns;
-  }
-
-  private String setAppInstanceValue() {
-    var propertyValue = ConfigProvider.getConfig().getValue("app.instance", String.class);
-    if (Objects.isNull(propertyValue) || propertyValue.isEmpty() || propertyValue.isBlank()) {
-      LOG.warn("property app.instance is not set, going with the default value");
-      propertyValue = "default";
+    private static final Logger LOG = LoggerFactory.getLogger(Operator.class);
+    private static final String MANAGED_BY = "managedBy";
+    private static final String WEBSITE = "website";
+    private static final String ENVIRONMENT = "environment";
+    private static final String SPASHIP = "spaship";
+    private final OpenShiftClient ocClient;
+    private final EventManager eventManager;
+    private final String domain;
+    private final String routerDomain;
+    private final String routerShard;
+    private final String appInstance;
+    private final String deDebugNs;
+
+    public Operator(@Named("default") OpenShiftClient ocClient,
+                    EventManager eventManager, @Named("deNamespace") String ns) {
+        this.ocClient = ocClient;
+        this.eventManager = eventManager;
+        domain = ConfigProvider.getConfig().getValue("operator.domain.name", String.class);
```

Comment:
@arkaprovob  can you use a variable instead of directly using the string?

`private static final String OPERATOR_DOMAIN_NAME = "operator.domain.name";`

Reply:
Sure @arkaprovob